lib/std/Target.zig

```diff
@@ -1140,6 +1140,13 @@ pub const Cpu = struct {
             };
         }
 
+        pub inline fn isLoongArch(arch: Arch) bool {
```

Comment:
This conflicts with my #20894; would you mind leaving this part out?

Reply:
Removed.